Tela de cadastro funcional e bonita

leoheisler opened the pull request:
(nao ta bonita -> agora ta bonita

leoheisler committed:
InicioCadastro

leoheisler committed:
Termino Front tela de Cadastro

leoheisler committed:
Connect to back

leoheisler committed:
aaaaaa

leoheisler committed:
postRequest solicitador

leoheisler committed:
debug nas mascaras

leoheisler committed:
Arrumando index

leoheisler committed:
Merge branch 'main' into Tela-Cadastro

marcelogborges reviewed (approved):
> Reviewed 3 of 3 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/Spolavore/connect-fix/3)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @Ignacio04, @Spolavore, and @Unfollow1)



<!-- Sent from Reviewable.io -->

Unfollow1 committed:
Melhorias Visuais e Adição de Campo Documento

adicionado campo "Comprovante de profissão" obrigatório, porém não capturado pelo userData, além de melhorias visuais à tela de cadastro

Unfollow1 commented:
pode aceitar 👍
